docs/man_pages/publishing/appmanager-upload-ios.md

```diff
@@ -3,7 +3,7 @@ appmanager upload ios
 
```

Comment:
Pretty much =) I am working on the source for the wp8 help though, I will send it shortly :) The android one is already posted above.